src/Components/Header/Header.css

```diff
@@ -35,6 +35,5 @@ a {
 }
 
 .active {
-  font-style: italic;
```

Comment:
I thought the italics were maybe distracting, so just left the active nav item with an underline instead.